lib/ood_core/version.rb

```diff
@@ -1,4 +1,4 @@
 module OodCore
   # The current version of {OodCore}
-  VERSION = "0.17.1"
+  VERSION = "0.17.6"
```

Comment:
Watch out for this. Seems you need to reset this file back to the master version.